cluster/manifests/kube-proxy/daemonset.yaml

```diff
@@ -28,6 +28,8 @@ spec:
         effect: NoExecute
       - operator: Exists
         key: CriticalAddonsOnly
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
```

Comment:
why?

Reply:
So the corresponding pod would be scheduled, see example at [link](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#how-daemon-pods-are-scheduled).

Reply:
ah ok, the config "sounded" exactly the opposite